Documentation/op-guide/clustering.md

```diff
@@ -214,17 +214,17 @@ There two methods that can be used for discovery:
 
 ### etcd discovery
 
-To better understand the design about discovery service protocol, we suggest reading the discovery service protocol [documentation][discovery-proto].
+To better understand the design of the discovery service protocol, we suggest reading the discovery service protocol [documentation][discovery-proto].
 
 #### Lifetime of a discovery URL
 
-A discovery URL identifies a unique etcd cluster. Instead of reusing a discovery URL, always create discovery URLs for new clusters.
+A discovery URL identifies a unique etcd cluster. Instead of reusing an existing discovery URL, a new cluster must create new discovery URLs.
```

Comment:
`Instead of reusing a discovery URL, always create a new discovery URL for a new cluster.`? clusters don't create discovery urls